ci/prow/e2e_prebuilt_kernel_module

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+echo "Label the node so that it gets the module..."
+kubectl label node minikube wants-module=kmm_ci_a
+
+echo "Build and deploy the controller image..."
+./ci/github/build_controller
+
+echo "Build and deploy the DriverContainer..." 
+./ci/github/build_driver_container
+
+echo "Deploy KMMO..."
+make deploy KUSTOMIZE_CONFIG_DEFAULT=ci/install-ci
+
+
+echo "Wait until the KMMO Deployment is Available..."
+kubectl wait --for condition=Available deployments.apps -n kmm-operator-system kmm-operator-controller-manager --request-timeout=1m
+
+echo "Describe the Deployment and get its YAML if that failed..."
+kubectl describe deployments.apps -n kmm-operator-system kmm-operator-controller-manager
+kubectl get -o yaml deployments.apps -n kmm-operator-system kmm-operator-controller-manager
+#        if: ${{ failure() }}
+
+echo "Check that the kmm_ci_a module is not loaded on the node..."
+          if minikube ssh -- lsmod | grep kmm_ci_a; then
+            echo "Unexpected lsmod output - the module should not be loaded"
+            exit 1
+          fi
+
+echo "Add an kmm-ci Module that contains a valid mapping..."
+          sed -e "s/KVER_CHANGEME/$(uname -r)/g" \
+            -e s/NAME_CHANGEME/kmm-ci/ \
+            -e s/KMOD_CHANGEME/kmm_ci_a/ \
+            ci/module-kmm-ci.template.yaml | tee module-kmm-ci.yaml
+          kubectl apply -f module-kmm-ci.yaml
+
+echo "Check that the module gets loaded on the node..."
+          until minikube ssh -- lsmod | grep kmm_ci_a; do
+            sleep 3
+          done
+
+echo "Check that the node gets labeled with the module's name..."
+          until kubectl get node minikube -o jsonpath='{.metadata.labels}' | jq -e 'has("kmm.node.kubernetes.io/kmm-ci.ready")'; do
+            sleep 3
+          done
+
+echo "Check that the daemon-set for device plugin is running..."
+          until kubectl get ds  -l 'kmm.node.kubernetes.io/module.name=kmm-ci,kmm.node.kubernetes.io/role=device-plugin' -ojson | jq -e '.items[] | select(.status.numberReady == 1)'; do 
+            sleep 3
+          done
```

Comment:
Why do we need to check it? if the daemonset doesn't exist then the kernel module won't be loaded.

I think we should only test input-output stuff and not be tightly coupled with the internal implementation (what if we stop using daemonsets for example?)

Reply:
This is to verify that the device plugin DaemonSet is running, I don't think there is any other way to make that check.

Reply:
Oh I missed the `for device plugin` in the description.
Let's keep it then.